components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/api/impl/GroupImpl.java

```diff
@@ -58,7 +61,9 @@
     public Response createGroup(DeviceGroup group) {
         String owner = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUsername();
         if (group == null) {
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/305/commits/c19ddcfe1549144e93e3781ec73bc8be1c6b2da3